nancy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:latest
+
+RUN apk update \
+    && apk upgrade \
+    && apk add --no-cache --update \
+       ca-certificates \
+       bash \
+       make \
+    && update-ca-certificates 2>/dev/null || true \
```

Comment:
```suggestion
    && ( update-ca-certificates 2>/dev/null || true ) \
```

Reply:
not sure I'd be happy about throwing away errors, either, but...

Reply:
> sure about that logic?

No 😝 In fact I don't think we need that entire line. I will fix it now.